app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -92,6 +94,48 @@ class WalkthroughTopicListFragmentTest {
     )
   }
 
+  // If progressBar is getting hidden on scrolling to the second row of items or not
```

Comment:
Unable to understand this statement, what does this mean?

Reply:
@rt4914 This does look incorrect, could you please suggest some proper comment for this?